osu.Game/Overlays/Settings/Sections/Gameplay/GeneralSettings.cs

```diff
@@ -76,6 +76,11 @@ private void load(OsuConfigManager config)
                 {
                     LabelText = "Score display mode",
                     Bindable = config.GetBindable<ScoringMode>(OsuSetting.ScoreDisplayMode)
+                },
+                new SettingsCheckbox
+                {
+                    LabelText = "Show playfield border",
+                    Bindable = config.GetBindable<bool>(OsuSetting.ShowPlayfieldBorder)
                 }
```

Comment:
Seeing that this is now a osu!-specific setting the location of it doesn't make much sense. This should be in `OsuSettingsSubsection`.